### PR TITLE
Add an indicator when events are live

### PR DESCRIPTION
**Pull request by snwessel**
@ltann and I paired on this one:

**Adding a little green dot to events when they are currently happening** on both the calendar and "Relevant" section. The dot only appears for events that have a defined start and end time. 

Also a couple of other minor tweaks including:
- When "Relevant" events have an undefined button text, the button is now hidden
- Updated from "class" to "className" in a couple of places to fix warnings 

image
image

### CODE REVIEW

File: src/components/RelevantRightNow.js [hunk 1]
```diff
@@ -50,7 +49,10 @@ function isTimeBetween(start, end) {
   // used different date to capture screenshots
   // using now means all will be filtered out
   const currentTime = new Date();
-  return isWithinInterval(currentTime, {
+  // return false if either the start or end is undefined
+  const range_defined = start !== undefined && end !== undefined;
```
Comment (nupur-n) [inline src/components/RelevantRightNow.js:53]: might be a design thing but im wondering if the cards now look awkward cause they're different sizes

**Review by nupur-n — APPROVED** [review]
just one comment about design ~ things ~ but coding wise LGTM.

File: src/components/RelevantRightNow.js (hunk 1 above)
Comment (snwessel) [reply src/components/RelevantRightNow.js:53]: That's a good point. I think ideally they'll all have buttons and descriptions, which will make them at least closer in size. 

Might be nice to have them all the same height too though if we have time.

**Review by cindy1u0 — APPROVED** [review]
Nice ~

**Pull request merged**